### DISCUSSION TITLE
How to print the path at which Assimp is looking for files?

### DISCUSSION THREAD

**Discussion by Gustan13** [assimp/assimp#5871] [Q&A] [answered]
I'm building a project in which I'm using Assimp, but the Importer class can't find any of the files. I'm using relative paths and I've already set my project's working directory, yet still no success. I'm kind of lost here, can someone help me?

**Comment by kimkulling** [comment, marked as answer]
If I get it right, you would like to learn which locations will be looked for your files, right? 
- The working directory and all relative paths from it
-  Any kind of absolute paths
